src/NUnitFramework/framework/Constraints/DictionaryContainsValueConstraint.cs

```diff
@@ -61,11 +61,13 @@ protected override bool Matches(IEnumerable actual)
         /// Flag the constraint to use the supplied predicate function
         /// </summary>
         /// <param name="comparison">The comparison function to use.</param>
+        /// <typeparam name="TActualValue">The type of the dictionary's <c>TValue</c>.</typeparam>
+        /// <typeparam name="TExpected">The type of the expected value.</typeparam>
         /// <returns>Self.</returns>
-        public DictionaryContainsValueConstraint Using<TCollectionType, TMemberType>(Func<TCollectionType, TMemberType, bool> comparison)
+        public DictionaryContainsValueConstraint Using<TActualValue, TExpected>(Func<TActualValue, TExpected, bool> comparison)
```

Comment:
As per comments in https://github.com/nunit/docs/pull/937 use `TActualValueElement`

Reply:
fixed